jwst/regtest/test_nirspec_ifu_spec2.py

```diff
@@ -50,7 +50,7 @@ def test_spec2(run_spec2, fitsdiff_default_kwargs, suffix):
 
 
 @pytest.fixture()
-def run_photom(jail, rtdata):
+def run_photom(tmp_cwd_module, rtdata):
```

Comment:
Not sure what is going on here.  one module-scoped fixture (`tmp_cwd_module`) and one function-scoped fixture ('rtdata') are being used simultaneously.  Would be good to sort out what is going on.  I can't run regtests over here.  ;-)

Reply:
looks to me like both the run_photom and run_extract1d fixtures should be module-scoped to permit multiple tests of their output without rerunning.  I passed just rtdata_module into these and all the tests in the file pass locally.